<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Global Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="cruiseControlBox">
       <property name="title">
        <string>Cruise Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Kp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="kpSlider">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="kpLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ki</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="kiSlider">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="kiLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Kd</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="kdSlider">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="kdLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Set Point:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SP"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Suspension</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>compression:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>damping:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="compression_FD"/>
            </item>
            <item row="1" column="4">
             <widget class="QDoubleSpinBox" name="compression_RP"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="damping_FD"/>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="compression_FP"/>
            </item>
            <item row="2" column="3">
             <widget class="QDoubleSpinBox" name="damping_RD"/>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="damping_FP"/>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="compression_RD"/>
            </item>
            <item row="2" column="4">
             <widget class="QDoubleSpinBox" name="damping_RP"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Front Driver&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>&lt;b&gt;Front Pass.&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>&lt;b&gt;Rear Driver&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;b&gt;Rear Pass.&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>stiffness:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="stiffness_FD"/>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="stiffness_FP"/>
            </item>
            <item row="3" column="3">
             <widget class="QDoubleSpinBox" name="stiffness_RD"/>
            </item>
            <item row="3" column="4">
             <widget class="QDoubleSpinBox" name="stiffness_RP"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Tires</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="grip_FP"/>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>&lt;b&gt;Rear Pass.&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QDoubleSpinBox" name="grip_RP"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="grip_FD"/>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="grip_RD"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Grip:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>&lt;b&gt;Front Pass.&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>&lt;b&gt;Rear Driver&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>&lt;b&gt;Front Driver&lt;/b&gt;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Steerable:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="steerable_FD">
              <property name="text">
               <string/>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QCheckBox" name="steerable_FP">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QCheckBox" name="steerable_RD">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QCheckBox" name="steerable_RP">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Roll influence:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="rollInfluence_FD"/>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="rollInfluence_FP"/>
            </item>
            <item row="2" column="3">
             <widget class="QDoubleSpinBox" name="rollInfluence_RD"/>
            </item>
            <item row="2" column="4">
             <widget class="QDoubleSpinBox" name="rollInfluence_RP"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Game</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Time Limit:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="timeEdit">
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Tilt Threshold:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QDial" name="dial">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="sliderPosition">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGraphicsView" name="graphicsView"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="simulateButton">
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Load Factory Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kpSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kpLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kiSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kiLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kdLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
